docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -21,10 +21,21 @@ const { buildNavBar } = require("./build-api-nav");
 const { renderAlertNode, renderBlockQuoteNode, renderTableNode } = require("./custom-renderers");
 const { createHugoFrontMatter } = require("./front-matter");
 
-const apiReportsDirectoryPath = path.resolve(__dirname, "..", "_api-extractor-temp", "_build");
-const apiDocsDirectoryPath = path.resolve(__dirname, "..", "content", "docs", "apis");
+/**
+ * To rendering multiple versions of the API docs, this function is called for each version.
+ * @param apiVersionNum api version number, e.g. "2.0"
```

Comment:
Nit:
```suggestion
 * @param {number} apiVersionNum - The release version for which API docs will be rendered.
```

It would also probably be good to call out how we organize the docs artifacts currently, since this functionality depends on that. E.g. `2.0` means we will read from `../api-extractor-temp/_build/2.0` and we will write to `content/docs/apis/2.0`.

Reply:
Updated